### PR TITLE
Fixed SMI Table horizontal scroll in mobile devices

### LINKED ISSUE #3826: Unable to horizontal scroll 'Service Mesh Interface Compliance' table in mobile device
### Description

While using a mobile device the Service Mesh Interface Compliance table isn't scrolling and we cannot see the columns to the right.
on the page [https://layer5.io/service-mesh-landscape](https://layer5.io/service-mesh-landscape)

### Expected Behavior

The table should scroll towards the right to view the rest of the columns.

### Screenshots

https://user-images.githubusercontent.com/88512639/221382563-9c90bd83-5acf-49e1-aa47-86945bea4c44.mp4

### Environment:
- Host OS: Android
- Browser: Chrome

I am working on this issue

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by AkashS20**
**Description**

This PR fixes #3826 
Added horizontal scroll in SMI Table and fixed orientation of legend rows on small devices

[Video](https://user-images.githubusercontent.com/92908347/231228036-79538e44-9378-4bc9-888d-a4f7f2cea5cc.webm)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c2f2c12ca52c557ba55dc30609170ebffa154ce7 at: https://64358f71c3a115088109333a--layer5.netlify.app

File: src/components/SMI-Table/SMITable.style.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import styled from "styled-components";
 
 export const TableWrapper = styled.div`
-overflow: hidden;
+overflow-x: auto;
```
Comment (rajiv-swipejobs) [inline src/components/SMI-Table/SMITable.style.js:4]: looks good to me. but the formatting of this style file is disturbed. it's not related to this change though

**Review by rajiv-swipejobs — APPROVED** [review]

File: src/components/SMI-Table/SMITable.style.js
Comment (rajiv-swipejobs) [inline src/components/SMI-Table/SMITable.style.js:1]: looks good to me. but the formatting of this style file is disturbed. it's not related to this change though

Comment (thoughtlessmind) [inline src/components/SMI-Table/SMITable.style.js:1]: looks good to me. but the formatting of this style file is disturbed. it's not related to this change though

**Review by thoughtlessmind — APPROVED** [review]

**Pull request merged**